### PR TITLE
Colab April 19 Update and Issue

### PR DESCRIPTION
**Pull request by Sewlell**
# Changelog
- Fix `ntlk` issue to settle down error when using Long TTS Sentence Mode.
- Fix `encodec_4cb2048_giga.th` downloading in the wrong place by just moving it automatically. ( ChatGPT moment )
- Fix `typer` dependency issue.
- Fix and adding instructions.

# Notable Issue
- That finetuned model of that 330M was said to trained on `pyannote.audio 0.0.1` and `torch 1.10.0+cu102`, whereas user is having `pyannote.audio 3.1.1` and `torch 2.2.1+cu121`. Saying it will have a bad time unless revert it. ( no sans intended )

# Attachment for the Notable Issue
```
Lightning automatically upgraded your loaded checkpoint from v1.5.4 to v2.2.2. To apply the upgrade to your files permanently, run `python -m pytorch_lightning.utilities.upgrade_checkpoint ../root/.cache/torch/whisperx-vad-segmentation.bin`
Model was trained with pyannote.audio 0.0.1, yours is 3.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.2.1+cu121. Bad things might happen unless you revert torch to 1.x.
```

Can still running everything normally beside possible degradation of quality.

By the way change everything inside that consist "VoiceCraft-gradio-colab" to "VoiceCraft". I gonna change my repository name sooner or later.

### CODE REVIEW

File: gradio_app.py [hunk 1]
```diff
@@ -101,6 +104,19 @@ def load_models(whisper_backend_name, whisper_model_name, alignment_model_name,
     if not os.path.exists(encodec_fn):
         os.system(f"wget https://huggingface.co/pyp1/VoiceCraft/resolve/main/encodec_4cb2048_giga.th")
 
+    # Define the source and destination paths
+    source_path = "/content/encodec_4cb2048_giga.th"
+    destination_path = "/content/VoiceCraft-gradio-colab/pretrained_models/encodec_4cb2048_giga.th"
```
Comment (jasonppy) [inline gradio_app.py:109]: /content/ is only suitable colab, could you also make it compatible for if people run it locally?

**Comment by jasonppy** [comment]
Thanks, what is the long tts error that you encountered?

File: gradio_app.py (hunk 1 above)
Comment (jasonppy) [reply gradio_app.py:109]: this issue is solved in https://github.com/jasonppy/VoiceCraft/pull/104

**Comment by Sewlell** [comment]
```
*** When using Long TTS Sentence mode.

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/gradio/queueing.py", line 407, in call_prediction
    output = await route_utils.call_process_api(
  File "/usr/local/lib/python3.10/dist-packages/gradio/route_utils.py", line 226, in call_process_api
    output = await app.get_blocks().process_api(
  File "/usr/local/lib/python3.10/dist-packages/gradio/blocks.py", line 1550, in process_api
    result = await self.call_function(
  File "/usr/local/lib/python3.10/dist-packages/gradio/blocks.py", line 1185, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/usr/local/lib/python3.10/dist-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.10/dist-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.10/dist-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.10/dist-packages/gradio/utils.py", line 661, in wrapper
    response = f(*args, **kwargs)
  File "/content/VoiceCraft/gradio_app.py", line 212, in run
    sentences = sent_tokenize(transcript.replace("\n", " "))
  File "/usr/local/lib/python3.10/dist-packages/nltk/tokenize/__init__.py", line 106, in sent_tokenize
    tokenizer = load(f"tokenizers/punkt/{language}.pickle")
  File "/usr/local/lib/python3.10/dist-packages/nltk/data.py", line 750, in load
    opened_resource = _open(resource_url)
  File "/usr/local/lib/python3.10/dist-packages/nltk/data.py", line 876, in _open
    return find(path_, path + [""]).open()
  File "/usr/local/lib/python3.10/dist-packages/nltk/data.py", line 583, in find
    raise LookupError(resource_not_found)
LookupError: 
**********************************************************************
  Resource punkt not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('punkt')
  
  For more information see: https://www.nltk.org/data.html

  Attempted to load tokenizers/punkt/PY3/english.pickle

  Searched in:
    - '/root/nltk_data'
    - '/usr/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************
```

Here is it. Sorry for late respond as I am busy on Saturday.

**Comment by jasonppy** [comment]
Apologies. I finally decided to manually change the code rather than merge with yours since there are too many conflicts

**Pull request closed without merging**